packages/SettingsProvider/src/com/android/providers/settings/SettingsProvider.java

```diff
@@ -4449,6 +4449,33 @@ private int onUpgradeLocked(int userId, int oldVersion, int newVersion) {
                     currentVersion = 183;
                 }
 
+                if (currentVersion == 183) {
```

Comment:
Can you change this to `currentVersion < 183`?

Reply:
Is there any particular reason for this change?